tests/cncf/kubernetes/hooks/test_kubernetes.py

```diff
@@ -49,7 +47,7 @@ async def test_kubernetes__load_config(
 
 
 @pytest.mark.asyncio
-@mock.patch("astronomer.providers.cncf.kubernetes.hooks.kubernetes_async.KubernetesHookAsync._load_config")
+@mock.patch("astronomer.providers.cncf.kubernetes.hooks.kubernetes.KubernetesHookAsync._load_config")
```

Comment:
why do we need to do this? What if a customer uses the old import path?

Reply:
I believe customers will be using only the operator/sensor/extractor, not the hook. WDYT?

Reply:
- Assume one of the PROD DAG of a customer has 

`import astronomer.providers.cncf.kubernetes.hooks.kubernetes_async.KubernetesHookAsync`
wouldnt their DAG break? 

- There is evidence of some customers using hook directly, one of our customers  uses TrinoHook directly. I don't know whether there is a possibility of other users doing the same. But it's always better to be on safe side.

- We'll need to redirect the user to use the latest path if we still want to take the risk..something like below where we warn the users to not use deprecated paths

` warnings.warn(
                "The import path parameter <> has been deprecated. Use the import path <> instead.",
                DeprecationWarning,
                stacklevel=2,
 )`

Reply:
cc @kaxil to comment on this

Reply:
yeah changes to hook is considered a breaking change. Add a backwards-compatible path with deprecation warning for old path

Reply:
thank you @phanikumv @kaxil for pointing it out, I'll change it accordingly.

Reply:
I have tried to deprecate it in c2d42a78f686c4342c9d48cfb8205acc90cbd85a commit. Please let me know how it is looking

Reply:
Perfect ! Can you test once by using the deprecated path and let me know the result